rts/lra/coordinator/pom.xml

```diff
@@ -99,6 +106,30 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.narayana.arjunacore</groupId>
+            <artifactId>arjunacore</artifactId>
```

Comment:
This dependency is a duplicate and was added by mistake (by my IDE) and needs removing.